File: amr15a.c
#include <stdio.h>
int main()
{
    short N, a, lucky = 0;
    scanf("%hd", &N);
    while(N--)
        {
            scanf("%hd", &a);
            (a%2 == 0)?(lucky++):(lucky--);
        }
    (lucky > 0)?(printf("READY FOR BATTLE")):(printf("NOT READY"));
    return 0;
}   

File: amr15a.java
import java.util.Scanner;
 
class Mahasena{
	public static void main(String[] arg){
		try{
			Scanner sc=new Scanner(System.in);
			byte n=sc.nextByte();
			byte l=0;
			while(n!=0){
				byte a=sc.nextByte();
				if(a%2==0)
					l++;
				else
					l--;
					n--;
			}
			if(l>0)
				System.out.println("READY FOR BATTLE");
			else	
				System.out.println("NOT READY");
		}catch(Exception e){}
	}
}  
